src/aliceVision/feature/apriltag/ImageDescriber_APRILTAG.cpp

```diff
@@ -0,0 +1,117 @@
+// This file is part of the AliceVision project.
+// Copyright (c) 2016 AliceVision contributors.
+// Copyright (c) 2012 openMVG contributors.
+// This Source Code Form is subject to the terms of the Mozilla Public License,
+// v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "ImageDescriber_APRILTAG.hpp"
+#include <aliceVision/gpu/gpu.hpp>
+#include <aliceVision/system/Logger.hpp>
+
+#include <apriltag/apriltag.h>
+#include <apriltag/tag16h5.h>
+#include <opencv2/core/core.hpp>
+
+namespace aliceVision {
+namespace feature {
+
+ImageDescriber_APRILTAG::AprilTagParameters::AprilTagParameters()
+{
+}
```

Comment:
```suggestion
ImageDescriber_APRILTAG::AprilTagParameters::AprilTagParameters() = default
```
U can even move it to the hpp